src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-pane/two-windings-transformer-pane.js

```diff
@@ -208,14 +190,14 @@ const TwoWindingsTransformerPane = ({
                     </h3>
                 </Grid>
             </Grid>
-            <Grid container spacing={2}>
+            <Grid container spacing={2} xs={8}>
```

Comment:
I think when you use xs, the grid should be <Grid container **item** spacing={2} xs={8}> or <Grid **item** spacing={2} xs={8}> or there will be warnings in the console

Reply:
Done, I went for the first option and applied it to all the places it's relevant